app/routes.py

```diff
@@ -1,2 +1,278 @@
-from flask import Blueprint
+from app import db
+from .models.task import Task
+from .models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime 
+import requests
+import os 
+
+task_bp = Blueprint("task", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goal", __name__, url_prefix="/goals")
+
+
+@task_bp.route("", methods=["POST"], strict_slashes=False)
+def create_task():
+    """
+        Input: Request to create a new task
+        Output: Returns an error if request body is missing attributes, otherwise, the new task
+                and its value is returned in json format
+    """
+    
+    request_body = request.get_json()
+    #checks to see if 'title' is found as a key in request_body called 'validation'
+    if 'title' not in request_body:
+        return make_response(jsonify({"details":"Invalid data"}), 400)
+    if 'description' not in request_body:
+        return make_response(jsonify({"details":"Invalid data"}), 400)
+    if 'completed_at' not in request_body:
+        return make_response(jsonify({"details":"Invalid data"}), 400)
```

Comment:
Just a minor suggestion but it would be typical in a case like this where the result is the same to write a compound `if <> or <> or` sort of statement to make it clear the resulting behavior is the same in all cases.